app/src/common/shared/org/mozilla/vrbrowser/ui/SettingsStore.java

```diff
@@ -73,4 +73,25 @@ public void setTelemetryEnabled(boolean isEnabled) {
         TelemetryHolder.get().getConfiguration().setCollectionEnabled(isEnabled);
     }
 
+
+    public void setCurrentSeesionId(int sessionId) {
+        SharedPreferences.Editor editor = mPrefs.edit();
+        editor.putInt(mContext.getString(R.string.settings_key_session_id), sessionId);
+        editor.commit();
+    }
+
+    public int getCurrentSessionId() {
+        return mPrefs.getInt(mContext.getString(R.string.settings_key_session_id), SessionStore.NO_SESSION_ID);
+    }
+
+    public void setPreviousSeesionId(int sessionId) {
```

Comment:
The setter can also be automatically done using the SessionChangeListener.onCurrentSessionChange listener insteaf of using external calls